examples/chip-tool/templates/commands.zapt

```diff
@@ -117,7 +125,7 @@ public:
 
         chip::Controller::{{asCamelCased parent.name false}}Cluster cluster;
         cluster.Associate(device, endpointId);
-        return cluster.{{asCamelCased name false}}(onSuccessCallback->Cancel(), onFailureCallback->Cancel(){{#chip_server_cluster_command_arguments}}, {{#if (isByteString type)}} reinterpret_cast<uint8_t*>(m{{asCamelCased label false}}), static_cast<uint32_t>(strlen(m{{asCamelCased label false}})){{else}}m{{asCamelCased label false}}{{/if}}{{/chip_server_cluster_command_arguments}});
+        return cluster.{{asCamelCased name false}}(onSuccessCallback->Cancel(), onFailureCallback->Cancel(){{#chip_server_cluster_command_arguments}}, {{#if (isByteString type)}} reinterpret_cast<char*>(m{{asCamelCased label false}}){{else}}m{{asCamelCased label false}}{{/if}}{{/chip_server_cluster_command_arguments}});
```

Comment:
Bytes seem to be more correctly represented as uint8_t than char  .

Should we not update the other way and enter a bug that 'using strlen for byte strings is not ok'?

Reply:
The byte string bit is going to get fixed in #5122 to some extent, right?  Though I guess https://github.com/project-chip/connectedhomeip/pull/5122#discussion_r587847807 is precisely about needing to change the member here to ByteSpan too.

And the rest of this looks like the PR I asked for in https://github.com/project-chip/connectedhomeip/pull/5122#discussion_r587847213 which I obviously approve of.  ;)